crates/client-api-messages/protobuf/client_api.proto

```diff
@@ -51,13 +51,15 @@ message IdentityToken {
 ///
 /// - `argBytes` is the arguments to the reducer, encoded as BSATN.
 ///
+/// - `requestId` is a identifier for client request.
 /// SpacetimeDB models reducers as taking a single `AlgebraicValue` as an argument, which
 /// generally will be a `ProductValue` containing all of the args (except the
 /// `ReducerContext`, which is injected by the host, not provided in this API).
 message FunctionCall {
     // TODO: Maybe this should be replaced with an int identifier for performance?
     string reducer = 1;
     bytes argBytes = 2;
+    bytes requestId = 3;
```

Comment:
@jdetter, pretty late but I was thinking, do we really need requestId as `bytes`? or something like `u64` would be sufficient?

Reply:
This definitely should just be a u64 or even just u32, but it would also break the ABI/API if we change it. I haven't looked at how its used, is there a reason other than breaking ABI/API why we can't just do u64 here?

Reply:
No, It's due to `oneOffQuery` uses bytes so to be consistent.
but now I feel to be right first, `OneOfQuery` has its own request and response message and doesn't collide with `TransactionUpdate` or `SubscriptionUpdate`. So We can leave that as it is until 0.9 and use `u32` for new messageId's I am adding. What do you say?